browser.js

```diff
@@ -1,4 +1,5 @@
 var createHmac = require('create-hmac')
+var MAX_INT = Math.pow(2, 30) - 1 // default in iojs
```

Comment:
I don't understand, the the maximum number in JS is (I guess) `2^53-1`, but how does this qualify for the maximum integer?

Also, how are we proving that keylen will never be > than this?